Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -186,12 +187,15 @@ private void OnFocus()
             #endregion  Interaction Constraint Setup
         }
 
-        public static void Show(MixedRealityControllerMapping controllerMapping, SerializedProperty interactionsList, Handedness handedness = Handedness.None)
+        public static void Show(MixedRealityControllerMapping controllerMapping, SerializedProperty interactionsList, Handedness handedness = Handedness.None, List<string> mappedControllers = null)
         {
             if (window != null)
             {
                 window.Close();
             }
+            
+            if (mappedControllers != null)
+                mappedControllerList = mappedControllers;
```

Comment:
Braces to match code style:

```suggestion
            if (mappedControllers != null)
            {
                mappedControllerList = mappedControllers;
            }
```